tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -62,24 +64,47 @@ def base_setup(
     @pytest.mark.usefixtures(base_setup.__name__)
     def test_pvc_creation_measurement_performance(self, teardown_factory, pvc_size):
         """
-        Measuring PVC creation time is less than 3 seconds
+        The test measures PVC creation times for sample volumes ( currently samples_num is 3)
+        makes sure create time is not greated than 3 seconds
+        calculates average creation time
+        and that the difference between creation time of each one of the PVCs and the average
+        is not more than 5%
```

Comment:
Pleas add to docstring explanation about the args. and empty line at the end

Reply:
@Avilir , thank you! I will add description of the arguments to the docstring.